docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -158,30 +158,42 @@ This allows you to develop in the same environment you will actually run your ap
 
 WARNING: Do not use this in production. This should only be used in a development environment. You should not run production application in dev mode.
 
-To do this you must have the `quarkus-undertow-websockets` extension installed:
+To do this you must build a 'mutable application', using the `fast-jar` format. Set the following properties in `application.xml`:
 
-[source,shell]
+[source,properties]
 ----
-./mvnw quarkus:add-extension -Dextensions="undertow-websockets"
+quarkus.package.type=fast-jar <1>
```

Comment:
How about if `mutable-application=true` we simply use the fast-jar and don't ask users to explicitly pick the type? At some point, I guess, we'll switch to the "fast jar" as the default anyway. At which point it'll be better to not bother with the legacy config option values.

Reply:
That's kinda problematic, as there is too much code that looks at the packaging type, and then all this code would also need to look at this additional property. I will just add a mutable-jar type to the packaging type.